formats/html_publication/publisher/details.json

```diff
@@ -21,6 +21,12 @@
     },
     "first_published_version": {
       "type": "boolean"
+    },
+    "isbn": {
+      "type": "string"
+    },
+    "print_meta_data_contact_address": {
```

Comment:
This corresponds to a field created in the `whitehall` project. Here is the migration for it: https://github.com/alphagov/whitehall/blob/072d107581c98a97f5d0a6eff6232821892f4099/db/migrate/20170223180546_add_metadata_fields_to_attachments.rb

I had a discussion with @h-lame  about what to name this: 
![screen shot 2017-03-09 at 15 32 41](https://cloud.githubusercontent.com/assets/1354439/23757516/fd888b5e-04de-11e7-9788-e768d348d2f7.png)

It can be renamed, but it's only being used for the print version. 

See the Trello card for why it was needed: https://trello.com/c/88zaXS8J/47-create-print-styles-for-official-documents